lua/nvim-tree.lua

```diff
@@ -200,14 +200,8 @@ function M.reset_highlight()
   renderer.render_hl(view.View.bufnr)
 end
 
-local prev_line
+-- removing `prev_line` since does not work when pressing `$` or `0`
 function M.place_cursor_on_node()
-  local l = vim.api.nvim_win_get_cursor(0)[1]
-  if l == prev_line then
-    return
-  end
-  prev_line = l
-
```

Comment:
this is used so the hijacking does not hijack when moving on the line. `$` and `0` were expected to be working normally.

Reply:
mm, I think hijacking `$` and `0` might be a nice addition, or at least make it optional

Reply:
Well if you have an overflowing filename, and you want to go to the end of the line, you should be able to move, thats why i implemented this. Better leave it like this for now, or it might create issues for other people.

Reply:
Removed